### ISSUE TITLE
"file does not exist [/opt/ros/indigo/share/nextage_moveit_config/config/kinematics.yaml]"

### ISSUE THREAD

**Issue by 130s** [tork-a/rtmros_nextage#193]
`moveit_config moveit_planning_execution.launch` terminates with the error:

```
$ roslaunch nextage_moveit_config moveit_planning_execution.launch 
... logging to /home/n130s/.ros/log/77f326e8-7529-11e5-a2db-f816542d218e/roslaunch-tork-kudu1-26978.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

error loading <rosparam> tag: 
    file does not exist [/opt/ros/indigo/share/nextage_moveit_config/config/kinematics.yaml]
XML is <rosparam command="load" file="$(find nextage_moveit_config)/config/kinematics.yaml"/>
The traceback for the exception was written to the log file
$

$ dpkg -p ros-indigo-nextage-moveit-config |grep Ver
Version: 0.6.4-0trusty-20151015-1727-+0000
```

**Comment by 130s** [comment]
This should be addressed in https://github.com/tork-a/rtmros_nextage/pull/192 (more specifically https://github.com/130s/rtmros_nextage/commit/851e84ffa247c475dfaf2e454057ff4b10386f9d).

**Comment by 130s** [comment]
For deb users, 
- most of you should not be affected by this issue. And you won't be affected either.
- only [Shadow repo](http://wiki.ros.org/ShadowRepository#Overview) users may be potentially affected. This should be resolved in 0.6.6 that will be soon available within in 2 or 3 days at most (release request is already in flight https://github.com/ros/rosdistro/pull/9667)
